.gitignore

```diff
@@ -52,3 +52,4 @@ config.bak
 testconfig.bak
 
 docs/site
+tmp.bats
```

Comment:
I assume this addition wasn't meant to be part of the PR?

I'm going to revert it due to checksuite being stuck on lint job and no apparent action actually running to restart it.

Reply:
Yes, this was not intended. GH actions experience degraded performance yesterday so I had to wait :)